contracts/dao-core/schema/dao-core.json

```diff
@@ -49,7 +49,7 @@
         ]
       },
       "initial_items": {
-        "description": "The items to instantiate this DAO with. Items are arbitrary key-value pairs who's contents are controlled by governance.\n\nIt is an error to provide two items with the same key.",
+        "description": "The items to instantiate this DAO with. Items are arbitrary key-value pairs whose contents are controlled by governance.\n\nIt is an error to provide two items with the same key.",
```

Comment:
for fixing this one, you'll need to modify this comment:

https://github.com/DA0-DA0/dao-contracts/blob/a3a45b9ae553529c526a4c01956fd673e7c684f7/contracts/dao-core/src/msg.rs#L46-L49

the code these json files is auto generated and CI will complain.